pkg/storage/common/common.go

```diff
@@ -71,27 +73,36 @@ func ValidateManifest(imgStore storageTypes.ImageStore, repo, reference, mediaTy
 	switch mediaType {
 	case ispec.MediaTypeImageManifest:
 		var manifest ispec.Manifest
+
+		// validate manifest
+		if err := schema.ValidatorMediaTypeManifest.Validate(bytes.NewBuffer(body)); err != nil {
```

Comment:
If the "err" says that there is only ONE error and that is missing layers, then we should still pass.

Reply:
See the "warning" in the comments. We want to support that case in `zot`